LICENSE

```diff
@@ -1,4 +1,5 @@
 Copyright (c) 2014 Joe Nelson
+Modified work Copyright 2015 Google Inc.
```

Comment:
There is no rush/immediate need for the docker config file, the PR can stay open until there is permission to remove this line.